src/redux/actions.js

```diff
@@ -688,6 +688,16 @@ export function setLoadflowResultFilter(filterTab, loadflowResultFilter) {
     };
 }
 
+export const LOADFLOW_RESULT_SORT = 'LOADFLOW_RESULT_SORT';
+
+export function setLoadflowResultSort(tab, loadflowResultSort) {
+    return {
+        type: LOADFLOW_RESULT_SORT,
+        tab: tab,
+        loadflowResultSort: loadflowResultSort,
+    };
+}
+
```

Comment:
I know it's done the same with filters, but it would be a bit better to have only one action instead of an action for each result type